tools/ci_build/rewriter.py

```diff
@@ -0,0 +1,221 @@
+'''comment out unused ops in cpu_execution_provider.cc by a set of models'''
```

Comment:
Please include license headers in all new files.

Reply:
Adding ...